koans/ArrayKoans.php

```diff
@@ -0,0 +1,100 @@
+<?php
+namespace PhpKoans;
+
+use PHPUnit\Framework\TestCase;
+
+defined('__') or define('__', null);
+
+class ArrayKoans extends TestCase
+{
+    /**
+     * @testdox Index 0 is the first element of an array
+     */
+    public function testGetTheValueOfAnArray()
```

Comment:
You can be more expecific on the tests names. Maybe 
testGetTheValueOfAnArray -> getTheValueOfTheFirstElementInAArray

With this type of name, comments can be removed.